src/components/Radio/RadioButtonsField.tsx

```diff
@@ -1,25 +1,30 @@
 import { Radio } from "@deskpro/deskpro-ui";
-import { CreateMeta } from "../../api/types/createMeta";
+import { Option } from "../../api/types/createMeta";
+import { FieldMeta } from "../../api/types/types";
 
 export const RadioButtonsField = ({
   meta,
   field,
   onChange,
 }: {
-  meta: CreateMeta["projects"]["0"]["issuetypes"]["0"]["fields"]["0"];
-  field: any;
-  onChange: (value: any) => void;
+  meta: FieldMeta;
+  field: Option["id"];
+  onChange: (value: Option["id"]) => void;
 }) => {
-  const allowedValues = meta?.allowedValues ?? [];
+  const allowedValues = meta?.allowedValues as Option[] ?? [];
```

Comment:
How is it proven that `allowedValues` is an array of `Option`, I assume the linter thinks it can be otherwise so how can we be confident it's not the case.

Reply:
We are confident because in the parent element, we check the type of the custom field.